### PR TITLE
The repository is missing a code of conduct

### PR DESCRIPTION
**Pull request by plehegar**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build ERRORED

Started: 2017-09-28 12:13:50
Finished: 2017-09-28 12:18:23


<h2>Failing Jobs</h2>
<ul>

<li>lint</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18947)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/280839738)

**Review by gsnedders — CHANGES_REQUESTED** [review]
AFAICT, most other repos just put CoC information in CONTRIBUTING.md. I'd also like a link to https://www.w3.org/Consortium/pwe/#Procedures.

**Comment by plehegar** [comment]
I'm actually trying to follow the recommendation from GitHub. The .md file is suggested by the Insights/Community tab.

File: CODE_OF_CONDUCT.md
Comment (gsnedders) [inline CODE_OF_CONDUCT.md]: I'd suggest something more like, "For details on who to contact, see [here]."

Comment (plehegar) [reply CODE_OF_CONDUCT.md]: done

**Comment by annevk** [comment]
This is great to see, but I wonder if the project should have its own CoC and enforcement, since it's not exclusively a W3C project.

> Any individual in the W3C community —including staff, members, invited experts, participants in W3C meetings, in W3C teleconferences, and participants in mailing lists— may call upon the W3C ombuds for assistance.

It's not clear how this statement applies to this project, which involves folks who could be considered to be outside that community by some. It also doesn't mention GitHub/IRC.

**Comment by annevk** [comment]
(And yeah, when adopted we should link this from the README and the docs too.)

**Comment by annevk** [comment]
Yeah, we had a chat on the IRC channel about the lack of formalized governance and you really need something for cases like this.

**Comment by plehegar** [comment]
as a side, we'll need to look at the LICENSE.md and figure out how to evolve it.

File: CODE_OF_CONDUCT.md
Comment (annevk) [inline CODE_OF_CONDUCT.md]: You need to list these people. I can't see this team. Also, even if I could see this team, how could I contact them privately?

Comment (plehegar) [reply CODE_OF_CONDUCT.md]: assuming we find the idea of using GH team workable, we'll need to create the team. We shouldn't list them in this file imho. Re contact, we could provide and add an email alias. I'm open to alternatives. @sideshowbarker already found one person for the team already.

Comment (annevk) [reply CODE_OF_CONDUCT.md]: I don't think a GitHub team works as not everyone is able to see them. You need to be a member of the organization to see teams.

Comment (plehegar) [reply CODE_OF_CONDUCT.md]: How about having the list of individuals in a file called moderators.md with contact info, and link to that?

Comment (annevk) [reply CODE_OF_CONDUCT.md]: That works.

Comment (plehegar) [reply CODE_OF_CONDUCT.md]: I realized creating a separate file seemed awkward, so I figured we might as well put the list in the CODE_OF_CONDUCT.md itself. One issue is that the GH list of WPT owners isn't public.
@foolip ? is there a public document listing the WPT owners somewhere we can point to? If not, it seems missing and we could have that under docs/ somewhere. wdyt?

Comment (foolip) [reply CODE_OF_CONDUCT.md]: Is this repo-level admins or org-level owners? We discussed on IRC having a meta repo with things like this at the org-level. To currently list all the people who have admin access to the repo would be a very long list.

**Comment by jugglinmike** [comment]
In [RFC 17 - Code of Conduct](https://github.com/web-platform-tests/rfcs/pull/17), the newly-formed WPT Core Team is considering this issue.

**Comment by foolip** [comment]
I merged https://github.com/web-platform-tests/wpt/pull/18950 into this branch. The branch is still very far behind so it can't trigger CI correctly, will merge in master to fix that.

File: CODE_OF_CONDUCT.md
Comment (foolip) [reply CODE_OF_CONDUCT.md]: Resolved by https://github.com/web-platform-tests/wpt/pull/18950. Thoughts on that can still be expressed in this PR or the RFC to establish the CoC however.

File: CODE_OF_CONDUCT.md
Comment (foolip) [inline CODE_OF_CONDUCT.md]: Just in case anyone else spots the `http:`, no https://citizencodeofconduct.org/ doesn't work.

**Review by gsnedders — DISMISSED** [review]
This is still blocked on https://github.com/web-platform-tests/rfcs/pull/17 anyway; r- because we need that to pass.

File: CODE_OF_CONDUCT.md
Comment (annevk) [reply CODE_OF_CONDUCT.md:15]: https://github.com/stumpsyn/policies/issues/8

Comment (foolip) [reply CODE_OF_CONDUCT.md:15]: Right now I can't load the site over HTTP either. It looks like this site isn't very reliable, https://github.com/rust-lang/rust/issues/57402 is a similar report.

I suggest we link either https://web.archive.org/web/20200213153345/http://citizencodeofconduct.org/#unacceptable-behavior or https://github.com/stumpsyn/policies/blob/master/citizen_code_of_conduct.md#4-unacceptable-behavior instead.

**Review by foolip — APPROVED** [review]
This is blocked by https://github.com/web-platform-tests/rfcs/pull/54 landing, but I've reviewed it, so making that explicit.

File: CODE_OF_CONDUCT.md
Comment (klausw) [inline CODE_OF_CONDUCT.md:27]: I don't know if this is covered elsewhere, but I think moderators should have discretion to skip the warning step for clearly abusive or extreme behavior, especially from people who may not even be community members. Warnings are appropriate for established community members, or new members making an honest mistake, but I think that drive-by trolling or off-topic spam could justify an immediate kick/ban with no prior warning. (This could still be appealed as noted below.)

Comment (stephenmcgruer) [reply CODE_OF_CONDUCT.md:27]: Thanks Klaus; I've made sure to note that in the proposed new CoC based on the contributors covenant (https://github.com/web-platform-tests/wpt/pull/23762), but if we reject that one and proceed with this one I will also edit it accordingly.

**Comment by stephenmcgruer** [comment]
Closing in favor of the new PR, https://github.com/web-platform-tests/wpt/pull/23762

**Pull request closed without merging**
